### DISCUSSION TITLE
Please update! Wrong parameter in donations.

### DISCUSSION THREAD

**Discussion by coccoinomane** [coccoinomane/crabada.py#117] [Announcements]
Hello snib-snibbers,

I have just realized that there was an outdated parameter in the rewards calculation 🤦‍♂️

Since the Swimmer migration, Crabada.py computes the TUS rewards as 81 times the CRA rewards (334.125 divided by 4.125).
This was necessary because on Swimmer the TUS rewards are sent via an internal transaction, and therefore not trackable.

However, the CRA rewards have recently tripled!

This means that for the last couple of days, Crabada.py was sending too much TUS in donations 

Thanks to @yigitest, I have just fixed the issue, so **please pull the latest version from master!**

If you were affected by the issue, **please let me know on Telegram**: I will send the extra rewards back to your wallet tomorrow, as soon as I read your messages. My Telegram handle is `coccoinomane`.

Thank you and sorry for the inconvenience 🙏
Cocco

**Comment by Mrooie** [comment]
Ha! I take it as the Universe keeping energy exchange in balance. :D
Thank you for being honest. My extra donations can stay with you. :)
Cheers!

**Reply by coccoinomane** [reply]
Haha, thank you for your kind words ❤️

Now, rewards are fixed. Going forward, we will rely on the Web2 endpoints to track rewards, so that if the team changes rewards again, errors like this one won't happen again:

- https://github.com/coccoinomane/crabada.py/issues/118

In the meanwhile, thank you for your understanding!

Cheers,
Cocco
